# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Sveinn í Felli <sv1@fellsnet.is>, 2010,2016-2017
# Sveinn í Felli <sv1@fellsnet.is>, 2010
msgid ""
msgstr ""
"Project-Id-Version: The Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-04-06 20:38+0200\n"
"PO-Revision-Date: 2017-06-01 01:27+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: Icelandic (http://www.transifex.com/otf/torproject/language/is/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: is\n"
"Plural-Forms: nplurals=2; plural=(n % 10 != 1 || n % 100 == 11);\n"

#: ../tails_installer/creator.py:100
msgid "You must run this application as root"
msgstr "Þú verður að vera rót/root til að keyra þetta forrit"

#: ../tails_installer/creator.py:146
msgid "Extracting live image to the target device..."
msgstr "Afþjappa keyrslukerfi á viðtökutæki..."

#: ../tails_installer/creator.py:153
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Skrifaði á tækið með %(speed)d MB/sek"

#: ../tails_installer/creator.py:183
msgid "Setting up OLPC boot file..."
msgstr "Set upp OLPC ræsiskrá..."

#: ../tails_installer/creator.py:302
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "Vandamál kom upp við að keyra eftirfarandi skipun: `%(command)s`.\nNákvæmari útlistun var skrifuð í '%(filename)s'."

#: ../tails_installer/creator.py:321
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "Yfirfer SHA1-gátsummu Live ISO keyrslukerfis..."

#: ../tails_installer/creator.py:325
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "Yfirfer SHA256-gátsummu Live ISO keyrslukerfis..."

#: ../tails_installer/creator.py:341
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Villa: SHA1-gátsumma Live ISO keyrslukerfisins er ógild. Þú getur keyrt þetta forrit með --noverify viðfanginu til að komast framhjá þessari öryggisprófun."

#: ../tails_installer/creator.py:347
msgid "Unknown ISO, skipping checksum verification"
msgstr "Óþekkt ISO-diskmynd, sleppi gátsummuprófun"

#: ../tails_installer/creator.py:358
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "Ekki nægilegt diskapláss á tækinu.\n%dMB ISO + %dMB yfirlögn > %dMB laust pláss"

#: ../tails_installer/creator.py:365
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "Bý til %sMB varanlega gagnageymslu"

#: ../tails_installer/creator.py:426
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "Tekst ekki að afrita %(infile)s yfir í %(outfile)s: %(message)s"

#: ../tails_installer/creator.py:440
msgid "Removing existing Live OS"
msgstr "Fjarlægi Live ISO keyrslukerfi"

#: ../tails_installer/creator.py:449 ../tails_installer/creator.py:460
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "Tekst ekki að keyra chmod %(file)s: %(message)s"

#: ../tails_installer/creator.py:453
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "Gat ekki fjarlægt skrá úr fyrra Live ISO keyrslukerfi: %(message)s"

#: ../tails_installer/creator.py:465
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "Gat ekki fjarlægt möppu úr fyrra Live ISO keyrslukerfi: %(message)s"

#: ../tails_installer/creator.py:513
#, python-format
msgid "Cannot find device %s"
msgstr "Get ekki fundið tækið %s"

#: ../tails_installer/creator.py:673
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "Get ekki skrifað á %(device)s, sleppi þessu."

#: ../tails_installer/creator.py:692
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "Sumar disksneiðar marktækisins %(device)s eru tengdar í skráakerfið. Þær verða aftengdar áður en uppsetningarferlið hefst"

#: ../tails_installer/creator.py:735 ../tails_installer/creator.py:959
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Óþekkt skráakerfi. Það gæti þurft að forsníða tækið."

#: ../tails_installer/creator.py:738 ../tails_installer/creator.py:962
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Óstutt skráakerfi: %s"

#: ../tails_installer/creator.py:755
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr ""

#: ../tails_installer/creator.py:759
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "Tókst ekki að tengja tæki: %(message)s"

#: ../tails_installer/creator.py:764
msgid "No mount points found"
msgstr "Engir tengipunktar fundust"

#: ../tails_installer/creator.py:775
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "Fer í unmount_device fyrir '%(device)s'"

#: ../tails_installer/creator.py:785
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "Aftengi tengd skráakerfi á '%(device)s'"

#: ../tails_installer/creator.py:789
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "Aftengi '%(udi)s' á '%(device)s'"

#: ../tails_installer/creator.py:799
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "Tengipunktur %s er til staðar eftir aftengingu"

#: ../tails_installer/creator.py:812
#, python-format
msgid "Partitioning device %(device)s"
msgstr "Sneiði geymslupláss tækisins %(device)s"

#: ../tails_installer/creator.py:890
#, python-format
msgid "Updating properties of system partition %(system_partition)s"
msgstr "Uppfæri eiginleika kerfisdisksneiðar %(system_partition)s"

#: ../tails_installer/creator.py:944
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "Óstutt tæki '%(device)s', tilkynntu um þessa villu."

#: ../tails_installer/creator.py:947
msgid "Trying to continue anyway."
msgstr "Reyni samt að halda áfram."

#: ../tails_installer/creator.py:956 ../tails_installer/creator.py:1349
msgid "Verifying filesystem..."
msgstr "Yfirfer skráarkerfi..."

#: ../tails_installer/creator.py:980
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "Get ekki breytt heiti á gagnarými: %(message)s"

#: ../tails_installer/creator.py:985 ../tails_installer/creator.py:1382
msgid "Installing bootloader..."
msgstr "Uppsetning ræsistjóra..."

#: ../tails_installer/creator.py:1011
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr ""

#: ../tails_installer/creator.py:1019 ../tails_installer/creator.py:1400
#, python-format
msgid "Removing %(file)s"
msgstr "Fjarlægi %(file)s"

#: ../tails_installer/creator.py:1131
#, python-format
msgid "%s already bootable"
msgstr "%s er þegar ræsanlegt"

#: ../tails_installer/creator.py:1151
msgid "Unable to find partition"
msgstr "Gat ekki fundið disksneið"

#: ../tails_installer/creator.py:1174
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "Forsníð %(device)s sem FAT32"

#: ../tails_installer/creator.py:1234
msgid "Could not find syslinux' gptmbr.bin"
msgstr ""

#: ../tails_installer/creator.py:1247
#, python-format
msgid "Reading extracted MBR from %s"
msgstr ""

#: ../tails_installer/creator.py:1251
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr ""

#: ../tails_installer/creator.py:1264 ../tails_installer/creator.py:1265
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "Frumstilli aðalræsifærslu (MBR) %s"

#: ../tails_installer/creator.py:1270
msgid "Drive is a loopback, skipping MBR reset"
msgstr "Drifið er hringtengt (loopback), sleppi endurstillingu á MBR-ræsigeira"

#: ../tails_installer/creator.py:1274 ../tails_installer/creator.py:1528
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "Reikna SHA1 gátsummuna fyrir %s"

#: ../tails_installer/creator.py:1299
msgid "Synchronizing data on disk..."
msgstr "Samræmi gögn á diski..."

#: ../tails_installer/creator.py:1341
msgid "Error probing device"
msgstr "Villa við að yfirfara tæki"

#: ../tails_installer/creator.py:1343
msgid "Unable to find any supported device"
msgstr "Fann ekki nein studd tæki"

#: ../tails_installer/creator.py:1353
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Gakktu úr skugga um að USB-lykillinn þinn sé tengdur og að hann sé forsniðinn með FAT-skráakerfi"

#: ../tails_installer/creator.py:1356
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Óstutt skráakerfi: %s\nTaktu öryggisafrit af USB-lyklinum og forsníddu hann með FAT-skráakerfi."

#: ../tails_installer/creator.py:1423
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "Tókst ekki að sækja Win32_LogicalDisk; win32com fyrirspurn skilaði engum niðurstöðum"

#: ../tails_installer/creator.py:1476
msgid "Cannot find"
msgstr "Get ekki fundið"

#: ../tails_installer/creator.py:1477
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr ""

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Óþekkt útgáfa: %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "Næ í %s..."

#: ../tails_installer/gui.py:204
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Villa: gat ekki sett merkingu eða fengið UUID-auðkenni tækisins. Get ekki haldið áfram."

#: ../tails_installer/gui.py:253
#, python-format
msgid "Installation complete! (%s)"
msgstr "Uppsetningu lokið! (%s)"

#: ../tails_installer/gui.py:258
msgid "Tails installation failed!"
msgstr ""

#: ../tails_installer/gui.py:353
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Aðvörun: Þetta tól verður að keyra sem kerfisstjóri (administrator). Til að gera það, hægri-smelltu á táknmyndina og opnaðu eiginleikana (Properties). Á samhæfniflipanum (Compatibility), hakaðu við að keyra þetta forrit sem kerfisstjóri (Run this program as an administrator)."

#: ../tails_installer/gui.py:365 ../tails_installer/launcher.py:31
msgid "Tails Installer"
msgstr ""

#: ../tails_installer/gui.py:440
msgid "No device suitable to install Tails could be found"
msgstr ""

#: ../tails_installer/gui.py:442
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr ""

#: ../tails_installer/gui.py:474
#, python-format
msgid "%(size)s %(label)s"
msgstr "%(size)s %(label)s"

#: ../tails_installer/gui.py:480
#, python-format
msgid "%(vendor)s %(model)s (%(details)s) - %(device)s"
msgstr "%(vendor)s %(model)s (%(details)s) - %(device)s"

#: ../tails_installer/gui.py:488
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "Tækið \"%(pretty_name)s\" er of lítið til að hægt sé að setja Tails upp á því (að minnsta kosti %(size)s GB eru nauðsynleg)."

#: ../tails_installer/gui.py:501
msgid "\"Install by cloning\""
msgstr ""

#: ../tails_installer/gui.py:503
msgid "\"Install from ISO\""
msgstr "\"Setja upp af ISO\""

#: ../tails_installer/gui.py:504
#, python-format
msgid ""
"It is impossible to upgrade the device %(pretty_name)s because it was not "
"created using Tails Installer. You should instead use %(action)s to upgrade "
"Tails on this device."
msgstr "Ómögulegt er að uppfæra tækið %(pretty_name)s því það var ekki útbúið með Tails-uppsetningarforritinu. Þú ættir frekar að nota %(action)s til að uppfæra Tails á þessu tæki."

#: ../tails_installer/gui.py:523
msgid "An error happened while installing Tails"
msgstr ""

#: ../tails_installer/gui.py:535
msgid "Refreshing releases..."
msgstr "Endurles útgáfur..."

#: ../tails_installer/gui.py:540
msgid "Releases updated!"
msgstr "Uppfærði útgáfur!"

#: ../tails_installer/gui.py:579
msgid "Installation complete!"
msgstr "Uppsetningu lokið!"

#: ../tails_installer/gui.py:580
msgid "Installation was completed."
msgstr ""

#: ../tails_installer/gui.py:625
msgid "Unable to mount device"
msgstr "Tókst ekki að tengja tæki"

#: ../tails_installer/gui.py:632 ../tails_installer/gui.py:660
msgid "Please confirm your device selection"
msgstr "Staðfestu val þitt á tæki"

#: ../tails_installer/gui.py:633
#, python-format
msgid ""
"You are going to install Tails on the %(size)s %(vendor)s %(model)s device "
"(%(device)s). All data on the selected device will be lost. Continue?"
msgstr "Þú ert að fara að setja Tails upp á %(size)s %(vendor)s %(model)s tækinu (%(device)s). Öll gögn á valda tækinu munu tapast. Halda áfram?"

#: ../tails_installer/gui.py:650
#, python-format
msgid ""
"You are going to upgrade Tails on the %(parent_size)s %(vendor)s %(model)s "
"device (%(device)s). Any persistent volume on this device will remain "
"unchanged. Continue?"
msgstr "Þú ert að fara að uppfæra Tails á %(parent_size)s %(vendor)s %(model)s tækinu (%(device)s). Öll gögn á valda tækinu munu haldast óbreytt. Halda áfram?"

#: ../tails_installer/gui.py:706
msgid "Download complete!"
msgstr "Niðurhali lokið!"

#: ../tails_installer/gui.py:710
msgid "Download failed: "
msgstr "Niðurhal mistókst: "

#: ../tails_installer/gui.py:711
msgid "You can try again to resume your download"
msgstr "Þú getur reynt aftur til að klára niðurhalið"

#: ../tails_installer/gui.py:719
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "Skráin sem þú valdir er ekki lesanleg. Lagaðu heimildir hennar eða veldu aðra skrá."

#: ../tails_installer/gui.py:725
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "Gat ekki notað völdu skrána. Það gæti gengið betur ef þú færir ISO-skrána á rót drifsins (dæmi: C:\\)"

#: ../tails_installer/gui.py:731
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s valið"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr ""

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr ""

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr ""

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr ""

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr ""

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr ""

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr ""

#: ../tails_installer/utils.py:119
msgid "Could not open device for writing."
msgstr ""

#: ../data/tails-installer.ui.h:1
msgid "Use existing Live system ISO:"
msgstr ""

#: ../data/tails-installer.ui.h:2
msgid "Select a distribution to download:"
msgstr ""

#: ../data/tails-installer.ui.h:3
msgid "Target Device:"
msgstr ""

#: ../data/tails-installer.ui.h:4
msgid "Install Tails"
msgstr "Setja upp Tails"

#: ../data/tails-installer-launcher.ui.h:1
msgid ""
"To run Tails Installer you need an ISO image which can be downloaded from "
"the Tails website: <a "
"href=\"https://tails.boum.org/download/\">https://tails.boum.org/download/</a>"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:2
msgid "Install"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:3
msgid "• Install Tails on a new USB stick."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:4
msgid "• The USB stick that you install on is formatted and all data is lost."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:5
msgid "Upgrade"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:6
msgid "• Upgrade a Tails USB stick to the version of an ISO image."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:7
msgid ""
"• The encrypted persistent storage of the Tails USB stick that you upgrade "
"is preserved."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:8
msgid ""
"Need help? Read the <a "
"href=\"https://tails.boum.org/doc/first_steps/installation/\">documentation</a>"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:9
msgid "Install by cloning"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:10
msgid ""
"• Install Tails on another USB stick by copying the Tails system that you "
"are currently using."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:11
msgid ""
"• The encrypted persistent storage of the Tails USB stick that you are "
"currently using is not copied."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:12
msgid "Upgrade by cloning"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:13
msgid ""
"• Upgrade another Tails USB stick to the same version of Tails that you are "
"currently using."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:14
msgid "Upgrade from ISO"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:15
msgid "• Upgrade another Tails USB stick to the version of an ISO image."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:16
msgid ""
"Need help? Read the <a "
"href=\"file:///usr/share/doc/tails/website/doc/first_steps/installation.en.html\">documentation</a>"
msgstr ""
